CDK/Linux/hello-world/README.md

```diff
@@ -110,7 +112,14 @@ Below are an example of the value in `ImageBuilderPipelineConfigurations` key th
 
 ### Managed Components for EC2 ImageBuilder
 
-The new component script must be provided before the new image building pipeline can be added. However, AWS provides large amount of managed components that can be easily used. They can be found from [List and view component details](https://docs.aws.amazon.com/imagebuilder/latest/userguide/component-details.html)
+The new component script must be provided before the new image building pipeline can be added. However, AWS provides large amount of managed components that can be easily used. They can be found from [List and view component details](https://docs.aws.amazon.com/imagebuilder/latest/userguide/component-details.html). To use the AWS managed components, you can specify the ARN of the AWS Managed Component in the `components` list in the Image Builder Pipeline Configuration.
+
+### Output of the Pipeline
+
+After an EC2 Image Builder image pipeline is established after deploying this stack with proper configuration, it will start automatically to build the EC2 AMI image based on the components specified in the configuration. The pipeline will take a while to complete and when it is completed, the ID of the new AMI image that is built can be view from the following two locations:
```

Comment:
1. Can you please replace "EC2 AMI image" and with simply "AMI"?
2. Can you replace "the ID of the new AMI image that is built can be view from the following two locations" with "the ID of the new AMI that is built can be found in the following locations:"

Reply:
Thanks. Fixed as you suggested